website/docs/api/Humanizer.DefaultFormatter.md

```
@@ -243,6 +243,35 @@ The localized duration phrase\.
 If [timeUnit](Humanizer.DefaultFormatter.md#Humanizer.DefaultFormatter.TimeSpanHumanize(Humanizer.TimeUnit,int,bool).timeUnit 'Humanizer\.DefaultFormatter\.TimeSpanHumanize\(Humanizer\.TimeUnit, int, bool\)\.timeUnit') is unsupported or [unit](Humanizer.DefaultFormatter.md#Humanizer.DefaultFormatter.TimeSpanHumanize(Humanizer.TimeUnit,int,bool).unit 'Humanizer\.DefaultFormatter\.TimeSpanHumanize\(Humanizer\.TimeUnit, int, bool\)\.unit') is negative\.
 
+<a name='Humanizer.DefaultFormatter.TimeSpanHumanizeWithFractionalSeconds(decimal,bool)'></a>
+
+## DefaultFormatter\.TimeSpanHumanizeWithFractionalSeconds\(decimal, bool\) Method
+
+Returns the localized representation of a seconds value\.
+
+```csharp
+public virtual string TimeSpanHumanizeWithFractionalSeconds(decimal seconds, bool toSymbols);
+```
+#### Parameters
```

markdownlint-cli2 (0.23.1)

[warning] Heading levels should only increment by one level at a time
Expected: h3; Actual: h4

(MD001, heading-increment)